### PR TITLE
[SMALLFIX] Run key-value examples via bin/tachyon

### PR DESCRIPTION
**Pull request by uronce-cc**

### CODE REVIEW

File: bin/tachyon [hunk 1]
```diff
@@ -118,26 +120,36 @@ function runTest {
   return ${RET}
 }
 
-function journalCrashTest {
```
Comment (apc999) [inline bin/tachyon]: is this test no more used?

File: bin/tachyon [hunk 2]
```diff
@@ -15,11 +15,13 @@ function printUsage {
   echo -e "  bootstrap-conf\t Generate a config file if one doesn't exist"
   echo -e "  tfs           \t Command line input for generic filesystem user client."
   echo -e "  loadufs       \t Load existing files in underlayer filesystem into Tachyon."
+  echo -e "  runClass      \t Run a Tachyon class with main method."
   echo -e "  runTest       \t Run a end-to-end test on a Tachyon cluster."
   echo -e "  runTests      \t Run all end-to-end tests on a Tachyon cluster."
   echo -e "  journalCrashTest\t Test the Master Journal System in a crash scenario. Try 'tachyon journalCrashTest -help' for more help."
   echo -e "                  \t NOTE: This command will stop the existing server and creates a new one!"
   echo -e "  readJournal   \t Read a Tachyon journal file from stdin and write a human-readable version of it to stdout."
+  echo -e "  keyValueStoreTest\t Test the key-value store operations."
```
Comment (apc999) [inline bin/tachyon]: how about `runKVTest`? to keep name convention consistent?

File: bin/tachyon (hunk 1 above)
Comment (apc999) [inline bin/tachyon]: oh found it.

Comment (uronce-cc) [inline bin/tachyon]: It's still there, below the modifications.

File: bin/tachyon (hunk 2 above)
Comment (uronce-cc) [inline bin/tachyon]: sgtm

Comment (uronce-cc) [inline bin/tachyon]: renamed

Comment (apc999) [inline bin/tachyon]: emm. we have `journalCrashTest`, maybe make this starting with run?
I am not very strong on it, feel free to do whatever you like

**Comment by apc999** [comment]
+1 LGTM

File: bin/tachyon (hunk 2 above)
Comment (uronce-cc) [inline bin/tachyon]: I made it to `runJournalCrashTest` for consistency.

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**